src/WebJobs.Script/Utility.cs

```diff
@@ -893,6 +893,11 @@ public static string GetSasTokenExpirationDate(string valueToParse, bool isAzure
             }
             else
             {
+                var isUri = Uri.IsWellFormedUriString(valueToParse, UriKind.Absolute);
+                if (!isUri)
+                {
+                    return null;
```

Comment:
Would returning null also lead to the issue you're describing in the PR description? If the method calling this method doing a null check anywhere?

Reply:
Yup so the null check is being done [here](https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script.WebHost/Health/TokenExpirationService.cs#L102). If the value returned from this utility function is null, we just end up skipping any logging for this environment variable